.github/workflows/build-test-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+
+on: push
+
+env:
+  artifact_path: /tmp/external-system-test-api.zip
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: Prepare
+      run: sudo apt-get install openjdk-11-jdk xvfb
+    - name: Setup Scala
+      uses: olafurpg/setup-scala@v10
+      with:
+        java-version: openjdk@1.11.0-2
+    - name: Build external-system-test-api.jar'
+      run: ./build-external-api-jar/run.sh
+    - name: SHA256
+      run: |
+        ls -l ${{ env.artifact_path }}
+        sha256sum ${{ env.artifact_path }}
+    - name: Create Release
+      if: startsWith(github.ref, 'refs/tags/external-system-test-api-v')
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: false
+        prerelease: false
+    - name: Upload Release Asset
+      if: startsWith(github.ref, 'refs/tags/external-system-test-api-v')
```

Comment:
It would be nice if the jar could be downloaded and inspected even if this step is not executed. 

This can be achieved using  the [`upload-artifact`](https://github.com/actions/upload-artifact) action.

Reply:
Yep, but I think anyone can do it in fork when preparing PR